eks/cluster-loader/cluster-loader.gotmpl

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cl-service
+automountServiceAccountToken: true
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: cl-service
+    namespace: default
+
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cl2
+spec:
+  template:
+    spec:
+      containers:
+      - name: clusterloader2
+        image: 197575167141.dkr.ecr.us-west-2.amazonaws.com/clusterloader2:latest
+        command: ["/clusterloader"]
+        args: {{ .TestParams }}
+        volumeMounts:
+        - name: config
+          mountPath: /etc/config
+      restartPolicy: Never
+      serviceAccountName: cl-service
+      volumes:
+        - name: config
+          configMap:
+            name: clusterloader2-config
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .ConfigMap.Metadata.Name }}
+data:
+  config.yaml: |
+    {{ index .ConfigMap.Data "config.yaml" }}
+  deployment.yaml: |
```

Comment:
This works for your usecase, but it isn't flexible to any other type of resources. Ideally deployment.yaml and config.yaml are populated from the map instead of hardcoded here. As I understand it, they're already in the map.

Check out https://golang.org/pkg/text/template/#hdr-Variables
```range $index, $element := pipeline```
You can do 
```
{{ range $file, $data := .ConfigMap.Data }}
   $file: | 
      $data
{{ end }}
```

Reply:
Smart fix! I was wondering how to do that. Thanks.